sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/federal-restrictions-integration/federal-restrictions-integration.scheduler.ts

```diff
@@ -0,0 +1,49 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { FedRestrictionProcessingService } from "@sims/integrations/esdc-integration";
+import { QueueService } from "@sims/services/queue";
+import { QueueNames } from "@sims/utilities";
+import { Job, Queue } from "bull";
+import { QueueProcessSummary } from "../../../models/processors.models";
+import { BaseScheduler } from "../../base-scheduler";
+import { ESDCFileResponse } from "../models/esdc";
+
+@Processor(QueueNames.FederalRestrictionsIntegration)
+export class FederalRestrictionsIntegrationScheduler extends BaseScheduler<void> {
+  constructor(
+    @InjectQueue(QueueNames.FederalRestrictionsIntegration)
+    schedulerQueue: Queue<void>,
+    queueService: QueueService,
+    private readonly processingService: FedRestrictionProcessingService,
```

Comment:
I noticed some "processors" classes declaring the "main service' as `processingService` and others as the services name, for instance, it would be `fedRestrictionProcessingService` for this one.
Due to the lack of consistency, I would like to request to stick with what we have been doing and inject the services using the service name.

Reply:
got it... actually... I copy and paste these from the deleted controllers